conceptual_model/civil-status.md

```diff
@@ -0,0 +1,20 @@
+Civil Status
+============
+
+[JSON Schema](schemas/civil-status.schema.json)
+
+
+Attributes
+----------
+
+| Name                   | Type     
+| -----------------------|----------
+| lastName               | [string](field-types.md#string-field)
+| firstName              | [string](field-types.md#string-field)
+| middleNames            | [string](field-types.md#string-field)
+| title                  | [string](field-types.md#string-field)
+| gender                 | [gender](field-types.md#gender-field)
+| birthDate              | [date](field-types.md#date-field)
+| birthPlace             | [string](field-types.md#string-field)
+| birthCountry           | [country](field-types.md#country-field)
```

Comment:
For _birthPlace_ and _birthCountry_, France Connect give the INSEE code only. Thus, the CMS provider should find the city/country name corresponding to fill these attributes.